packages/artillery-tests/.env.example

```diff
@@ -5,4 +5,8 @@ MEDPLUM_PASSWORD=password
 MEDPLUM_URL=https://domain.com
 
 HAPI_URL=https://domain.com
-HAPI_TENANT_ID=some-id
\ No newline at end of file
+HAPI_TENANT_ID=some-id
+
+OSS_STAGING_URL=https://domain.com
+OSS_STAGING_API_KEY=some-key
+OSS_STAGING_FACILITY_ID=some-id
```

Comment:
Why "staging" on the env var name? We represent that with diff `.env` files, one for each env.